### DISCUSSION TITLE
Newly bookmarked manuals

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#79] [General]
Just dropped Volume 1 of the NOS 2 Reference Set.

https://github.com/KC5VDJ/CDC-Manuals

Volumes 2 and 3 in progress, Section 7 on both..

![Acrobat_aXVuRJcgSa](https://github.com/kej715/DtCyber/assets/136194499/771689c7-0d34-432c-9c09-984e968835cb)
![Acrobat_NBPB0Remrl](https://github.com/kej715/DtCyber/assets/136194499/2b473e24-e401-472d-9a39-6334ab90b8ab)

I'll use this thread to announce future drops here.

Also working on some protocols, as well as a pi interface, and will probably have a printer burster by the end of the week as time allows.  (Initially txt only, will add pdf and ps later).

**Comment by KC5VDJ** [comment]
Volume 2, Guide to System Usage is now up as well.  Still grinding away on Volume 3.

**Comment by KC5VDJ** [comment]
Errors are being "corrected" as-noticed as well...

![Acrobat_ykRLgoc4MW.png](https://github.com/kej715/DtCyber/assets/136194499/44e09f9f-7e09-418a-a05a-41964d7a13aa)

**Comment by drboone** [comment]
Er, is the intent to delete stuff frequently?

**Reply by KC5VDJ** [reply]
Huh?  If you mean the repository, no.  I only upload finished ones, otherwise it would get messy.

**Comment by drboone** [comment]
Oh, I see.  Some of the renames weren't detected by git as renames, so they arrived as delete + create.

**Reply by KC5VDJ** [reply]
Oh, I see.  Yeah, a minor naming issue had to be corrected to comply with the naming convention.
